### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by andymsuk**

### CODE REVIEW

File: components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export default function Footer() {
+  return (
+    <footer className="footer">
+      <div className="gel-wrap">
+        <div className="footer__social">
+          <div className="footer__social__item">
+            <a
+              href="https://www.facebook.com/roseatecards"
+              target="_blank"
+              rel="noopener noreferrer"
```
Comment (vanitabarrett) [inline components/Footer.tsx]: Think we need to add `rel="external"` to indicate that "the referenced document is not part of the same site as the current document" (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel)

File: components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+export default function Footer() {
+  return (
+    <footer className="footer">
+      <div className="gel-wrap">
+        <div className="footer__social">
+          <div className="footer__social__item">
+            <a
+              href="https://www.facebook.com/roseatecards"
+              target="_blank"
```
Comment (vanitabarrett) [inline components/Footer.tsx]: The w3c recommends against opening link in new tabs where possible (https://www.w3.org/TR/WCAG20-TECHS/G200.html)

File: components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+              className="footer__link footer__link--social"
+            >
+              Facebook
+          </a>
+          </div>
+          <div className="footer__social__item">
+            <a
+              href="https://www.instagram.com/roseatecards/"
+              target="_blank"
+              rel="noopener noreferrer"
```
Comment (vanitabarrett) [inline components/Footer.tsx]: Same comments apply here about opening in new tab and adding `rel="external"`

File: components/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+              className="footer__link footer__link--social"
```
Comment (vanitabarrett) [inline components/Footer.tsx]: Why is the `footer__link--social` class needed when it's wrapped in `footer__social__item` already?

File: pages/_app.tsx [hunk 5]
```diff
@@ -12,10 +13,13 @@ function MyApp({ Component, pageProps }: AppProps) {
         <title>Roseate Cards</title>
         <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet" />
```
Comment (vanitabarrett) [inline pages/_app.tsx:14]: Can we add a todo/Github issue to add meta descriptions and things for social media link sharing?

**Review by vanitabarrett — CHANGES_REQUESTED** [review]

File: components/Footer.tsx (hunk 1 above)
Comment (andymsuk) [reply components/Footer.tsx]: Done.

File: components/Footer.tsx (hunk 2 above)
Comment (andymsuk) [reply components/Footer.tsx]: Done.

File: components/Footer.tsx (hunk 4 above)
Comment (andymsuk) [reply components/Footer.tsx]: Changed.

File: components/Footer.tsx (hunk 3 above)
Comment (andymsuk) [reply components/Footer.tsx]: Done.

File: pages/_app.tsx (hunk 5 above)
Comment (andymsuk) [reply pages/_app.tsx:14]: Do you want to add it to your notes?

**Review by vanitabarrett — APPROVED** [review]

**Pull request merged**